### PR TITLE
updated index.ejs to pull users discord profile image and fill the user card

### PR DESCRIPTION
**Pull request by X-AmpleDevelopment**
updated index.ejs to pull the users discord profile image down from discords oauth.

end user will need to update there index.js to include the following:

                index: async ({ req, res, config }) => {
                    // Get the username and avatar URL from the session, with a default value
                    let username = req.session?.user?.username || "Guest";
                    let avatar = req.session?.user?.avatar
                        ? `https://cdn.discordapp.com/avatars/${req.session.user.id}/${req.session.user.avatar}.png`
                        : 'https://cdn.discordapp.com/embed/avatars/0.png';
                    const cards = [
                        {
                            title: "Server count",
                            icon: "single-02",
                            getValue: `${client.guilds.cache.size} out of 100`,
                            progressBar: {
                                enabled: false,
                                getProgress: (client.guilds.cache.size / 100) * 100
                            }
                        },
                        {
                            title: "CPU",
                            icon: "single-02",
                            getValue: os.cpus()[0].model.replace('(R) Core(TM) ', ' ').replace(' CPU ', '').split('@')[0],
                            progressBar: {
                              enabled: false,
                              getProgress: 50, // 0 - 100 (get a percentage of the progress)
                            },
                          },
                        {
                            title: "System Platform",
                            icon: "single-02",
                            getValue: os.platform(),
                            progressBar: {
                                enabled: false,
                                getProgress: 50
                            }
                        },
                        {
                            title: "Current User",
                            icon: avatar, // Use the avatar URL here
                            getValue: username,
                            progressBar: {
                                enabled: false,
                                getProgress: client.guilds.cache.size // Placeholder value
                            }
                        },
                    ];

### CODE REVIEW

**Review by mohaidarus — APPROVED** [review]
Looks good!

File: views/index.ejs [hunk 1]
```diff
@@ -70,11 +70,14 @@
                                     <% } %>
                                 </div>
                                 <div class="col-4 text-end">
-                                    <div class="icon icon-shape bg-gradient-primary shadow text-center border-radius-md">
-                                        <% if (stats.icon.startsWith('fa-')) { %>
-                                            <i class="<%= stats.icon %> text-lg opacity-10" aria-hidden="true"></i>
-                                        <% } else { %>
-                                            <i class="ni ni-<%= stats.icon %> text-lg opacity-10" aria-hidden="true"></i>
+                                    <div class="icon icon-shape text-center border-radius-md"
```
Comment (DJKnaeckebrot) [inline views/index.ejs:73]: might wanna readd "bg-gradient-primary"

File: views/index.ejs [hunk 2]
```diff
@@ -70,11 +70,14 @@
...
+                                         style="<% if (stats.icon.startsWith('http')) { %>background-image: url('<%= stats.icon %>'); background-size: cover; background-position: center;<% } else { %>background: linear-gradient(87deg, #11cdef, #1171ef);<% } %>">
```
Comment (DJKnaeckebrot) [inline views/index.ejs:74]: if we readd above this background: linear-gradient(87deg, #11cdef, #1171ef) is not neccesarry

**Review by DJKnaeckebrot — COMMENTED** [review]
@iMidnights please review aswell :)

File: views/index.ejs (hunk 1 above)
Comment (mohaidarus) [reply views/index.ejs:73]: There should be no background when using a custom image

Comment (DJKnaeckebrot) [reply views/index.ejs:73]: okay coolio great work than @X-AmpleDevelopment :)

Comment (X-AmpleDevelopment) [reply views/index.ejs:73]: Thanks both!

**Pull request merged**